src/Core/AdminConsole/Services/Implementations/WebhookIntegrationHandler.cs

```
@@ -29,7 +31,7 @@ public override async Task<IntegrationHandlerResult> HandleAsync(IntegrationMess
             case HttpStatusCode.ServiceUnavailable:
             case HttpStatusCode.GatewayTimeout:
                 result.Retryable = true;
-                result.FailureReason = response.ReasonPhrase;
+                result.FailureReason = response.ReasonPhrase ?? string.Empty;
```

❓ Should this and the same thing below return a more "default" failure reason instead of an empty string?

---

Good point. I changed it in the next commit to at least provide us with the status code:

``` csharp
    result.FailureReason = response.ReasonPhrase ?? $"Failure with status code {(int)response.StatusCode}";
```